fastlane/lib/fastlane/actions/cocoapods.rb

```diff
@@ -15,7 +15,9 @@ def self.run(params)
         end
 
         cmd << ['bundle exec'] if params[:use_bundle_exec] && shell_out_should_use_bundle_exec?
-        cmd << ['pod install']
+        cmd << ['pod']
+        cmd << [" _#{params[:pod_version]}_"] if params[:pod_version]
```

Comment:
I didn't know this syntax existed, but it seems to work. Is it something cocoapods specific or is it something that rubygems does?